aiidalab_widgets_base/databases.py

```diff
@@ -105,3 +105,7 @@ def _on_select_structure(self, change):
         self.structure = selected['cif'].get_ase()
         struct_url = selected['url'].split('.cif')[0] + '.html'
         self.link.value = '<a href="{}" target="_blank">COD entry {}</a>'.format(struct_url, selected['id'])
+
+    @default('structure')
+    def _default_structure(self):  # pylint: disable=no-self-use
```

Comment:
I believe that if you have `allow_none=True`, then this is not necessary, but there is probably also no harm in keeping this.

Reply:
ok, so we keep it, right?